sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_internal/algorithms/aes_kw.py

```diff
@@ -59,7 +60,7 @@ class AesKw192(_AesKeyWrap):
 
 class AesKw256(_AesKeyWrap):
     _key_size = 256
-    _name = "A256KW"
+    _name = KeyWrapAlgorithm.aes_256
```

Comment:
Was it intentional to only use an enum value for this class name, and not for `AesKw128` or `AesKw192`? It looks like all encryption and signature algorithm names were added to `_enums.py`.

Reply:
Yes; the versions of Key Vault we presently support only support A256KW, so that's the only one of these in `KeyWrapAlgorithm`. The others will come with 7.2-preview (#13561).